src/services/limits/limitsService.ts

```
@@ -43,19 +43,16 @@ export async function checkDepositLimits(
   organizationId: string | null,
 ): Promise<void> {
   const config = await getLimitConfig(audience);
-  const since24h = new Date(Date.now() - 24 * 60 * 60 * 1000);
-  const startOfMonth = new Date(
-    new Date().getFullYear(),
-    new Date().getMonth(),
-    1,
-  );
+  const now = new Date();
+  const startOfDay = new Date(Date.UTC(now.getUTCFullYear(), now.getUTCMonth(), now.getUTCDate()));
```

ESLint

[error] Replace `Date.UTC(now.getUTCFullYear(),·now.getUTCMonth(),·now.getUTCDate())` with `⏎····Date.UTC(now.getUTCFullYear(),·now.getUTCMonth(),·now.getUTCDate()),⏎··`

(prettier/prettier)